mlflow/projects/backend/local.py

```diff
@@ -142,12 +147,24 @@ def run(
                     if project.env_config_path
                     else _PythonEnv()
                 )
-            python_bin_path = _install_python(python_env.python)
-            env_root = _get_mlflow_virtualenv_root()
+
+            if is_in_databricks_runtime():
+                nfs_tmp_dir = get_or_create_nfs_tmp_dir()
+                envs_dir = Path(nfs_tmp_dir) / "envs"
+                pyenv_root = envs_dir / _PYENV_ROOT_DIR
+                virtualenv_root = envs_dir / _VIRTUALENV_ENVS_DIR
+                env_vars = _get_virtualenv_extra_env_vars(str(envs_dir))
+            else:
+                pyenv_root = None
+                virtualenv_root = Path(_get_mlflow_virtualenv_root())
+                env_vars = None
```

Comment:
Nit: The part is very similar to some code in `_get_or_create_virtualenv`, can we reuse that function ?

Reply:
Sounds good

Reply:
@WeichenXu123 I'll do refactoring in a follow-up PR.